src/test/java/RoundTripSearchTest.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import pages.MainPage;
+import pages.FlightSearchPage;
+
+import static org.testng.Assert.assertTrue;
+
+public class RoundTripSearchTest {
+    private WebDriver driver;
+    private MainPage mainPage;
+    private FlightSearchPage flightSearchPage;
+
+    @BeforeClass
+    public void setup() {
+        driver = new ChromeDriver();
```

Comment:
Setting driver and navigating to the app is common for all test classes, right?
Does it mean we can create a BaseTest class and do that actions there?